### PR TITLE
Remote jetpack install

### PR DESCRIPTION
**Pull request by planarvoid**
- add rest client and store for Jetpack to enable remote install
- add interface in both standard event driven arch and coroutines
- bump kotlin
- bump coroutines

### CODE REVIEW

File: example/src/test/java/org/wordpress/android/fluxc/store/JetpackStoreTest.kt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @Mock private lateinit var site: SiteModel
+    private lateinit var jetpackStore: JetpackStore
+
+    @Before
+    fun setUp() {
+        jetpackStore = JetpackStore(jetpackRestClient, dispatcher)
+    }
+
+    @Test
+    fun `on install triggers rest client and returns success`() = runBlocking {
```
Comment (0nko) [inline example/src/test/java/org/wordpress/android/fluxc/store/JetpackStoreTest.kt]: Cool method names :D

**Comment by 0nko** [comment]
We can :shipit: once the checks are green.

File: example/src/test/java/org/wordpress/android/fluxc/store/JetpackStoreTest.kt (hunk 1 above)
Comment (planarvoid) [reply example/src/test/java/org/wordpress/android/fluxc/store/JetpackStoreTest.kt]: not sure we want to write them like that but I think it's more readable for tests 😄

**Comment by 0nko** [comment]
:shipit:

**Pull request merged**